src/utils/typography.js

```diff
@@ -10,119 +10,138 @@ export const primaryFont = "tondo, sans-serif";
 //     button: '1.35em'
 // };
 
-
 export const H1 = styled.h1`
```

Comment:
Naming shouldn't be H1 but Title or Header or something more relevant.

Reply:
done